### ISSUE TITLE
pagination perPage calculation wrong

### ISSUE THREAD

**Issue by ackratos** [tendermint/tendermint#3065]
https://github.com/tendermint/tendermint/blob/1e1ca15bccedc16c2452c73745c3c2b5df91ae96/rpc/core/pipe.go#L153

```
func validatePerPage(perPage int) int {
	if perPage < 1 || perPage > maxPerPage {
		return defaultPerPage
	}
	return perPage
}
```

if perPage > maxPerPage we should return `maxPerPage` rather than `defaultPerPage`.

This is related with https://github.com/cosmos/cosmos-sdk/pull/2894
And we are talking about increase the perPage to 100 to 200 there

**Comment by fedekunze** [comment]
++

**Comment by ebuchman** [comment]
Fixed in #3124, will ship in v0.29.0. Let's open a new issue if we need to up the max.
